test/com/puppetlabs/http/client/impl/metrics_unit_test.clj

```diff
@@ -216,36 +213,35 @@
         (is (thrown? ExceptionInfo (metrics/get-client-metrics-data nil)))
         (is (thrown? IllegalArgumentException (Metrics/getClientMetrics nil)))
         (is (thrown? IllegalArgumentException (Metrics/getClientMetricsData nil))))
-      (testing (str "getClientMetrics|Data returns map with empty arrays as values"
-                    " if no requests have been made yet")
-        (is (= {"url" [] "url-and-method" [] "metric-id" []}
-               (into {} (Metrics/getClientMetrics (MetricRegistry.)))
-               (into {} (Metrics/getClientMetricsData (MetricRegistry.)))))
-        (is (= {:url [] :url-and-method [] :metric-id []}
-               (metrics/get-client-metrics (MetricRegistry.))
-               (metrics/get-client-metrics-data (MetricRegistry.)))))
+      (testing (str "getClientMetrics|Data returns data structure with empty arrays"
+                    "  as values if no requests have been made yet")
+        (let [empty-metrics (Metrics/getClientMetrics (MetricRegistry.))
+              empty-metrics-data (Metrics/getClientMetricsData (MetricRegistry.))]
+          (is (empty? (.getUrlTimers empty-metrics)))
+          (is (empty? (.getUrlAndMethodTimers empty-metrics)))
+          (is (empty? (.getMetricIdTimers empty-metrics)))
+          (is (empty? (.getUrlData empty-metrics-data)))
+          (is (empty? (.getUrlAndMethodData empty-metrics-data)))
+          (is (empty? (.getMetricIdData empty-metrics-data)))))
```

Comment:
I'm not sure it matters a ton, but this has deleted the tests for the clojure methods - I'm not sure that there's anywhere else that this specific case is being tested for them.